media-tv/mythtv/mythtv-30.0_p20190808.ebuild

```diff
@@ -0,0 +1,388 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+
+BACKPORTS="5cde0578d84926171b20c8f7e95a101e9b0b9457" # August 8, 2019
+
+MY_P=${P%_p*}
+MY_PV=${PV%_p*}
+
+inherit eutils flag-o-matic python-single-r1 qmake-utils readme.gentoo-r1 systemd vcs-snapshot
+
+MYTHTV_BRANCH="fixes/${P%.*}"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-respect_LDFLAGS.patch
+)
+
+DESCRIPTION="Open Source DVR and media center hub"
+HOMEPAGE="https://www.mythtv.org"
+SRC_URI="https://github.com/MythTV/mythtv/archive/${BACKPORTS}.tar.gz -> ${PF}.tar.gz"
+
+LICENSE="GPL-2+"
+KEYWORDS="~amd64 ~x86"
+SLOT="0/${PV}"
+
+IUSE_INPUT_DEVICES="input_devices_joystick"
+IUSE_VIDEO_CAPTURE_DEVICES="v4l ivtv ieee1394 hdpvr hdhomerun vbox ceton"
+IUSE="alsa altivec asi autostart bluray cdda cdr cec crystalhd debug dvd dvb egl exif fftw jack java
+	+lame lcd libass lirc +opengl oss perl pulseaudio python raw systemd vaapi vdpau vpx
+	+wrapper x264 x265 +xml xmltv +xvid +X zeroconf
+	${IUSE_INPUT_DEVICES} ${IUSE_VIDEO_CAPTURE_DEVICES}"
+REQUIRED_USE="
+	python? ( ${PYTHON_REQUIRED_USE} )
+	bluray? ( xml )
+	cdr? ( cdda )
+"
+
+COMMON="
+	acct-user/mythtv
+	dev-libs/glib:2
+	dev-libs/lzo
+	dev-qt/qtcore:5
+	dev-qt/qtdbus:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
+	opengl? ( dev-qt/qtopengl:5 )
+	dev-qt/qtscript:5
+	dev-qt/qtsql:5[mysql]
+	dev-qt/qtwebkit:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtxml:5
+	media-gfx/exiv2:=
+	media-libs/freetype:2
+	media-libs/libsamplerate
+	media-libs/taglib
+	lame? ( >=media-sound/lame-3.93.1 )
+	sys-libs/zlib
+	virtual/mysql
+	opengl? ( virtual/opengl )
+	X? (
+		x11-libs/libX11:=
+		x11-libs/libXext:=
+		x11-libs/libXinerama:=
+		x11-libs/libXrandr:=
+		x11-libs/libXv:=
+		x11-libs/libXxf86vm:=
+		x11-misc/wmctrl:=
+	)
+	alsa? ( >=media-libs/alsa-lib-1.0.24 )
+	bluray? (
+		media-libs/libbluray:=
+		dev-libs/libcdio:=
+		sys-fs/udisks:2
+	)
+	cec? ( dev-libs/libcec )
+	dvd? (
+		dev-libs/libcdio:=
+		sys-fs/udisks:2
+	)
+	egl? ( media-libs/mesa[egl] )
+	fftw? ( sci-libs/fftw:3.0=[threads] )
+	hdhomerun? ( media-libs/libhdhomerun )
+	ieee1394? (
+		>=media-libs/libiec61883-1.0.0
+		>=sys-libs/libavc1394-0.5.3
+		>=sys-libs/libraw1394-1.2.0
+	)
+	jack? ( media-sound/jack-audio-connection-kit )
+	lcd? ( app-misc/lcdproc )
+	libass? ( >=media-libs/libass-0.9.11:= )
+	lirc? ( app-misc/lirc )
+	perl? (
+		>=dev-perl/libwww-perl-5
+		dev-perl/DBD-mysql
+		dev-perl/HTTP-Message
+		dev-perl/IO-Socket-INET6
+		dev-perl/LWP-Protocol-https
+		dev-perl/Net-UPnP
+	)
+	pulseaudio? ( media-sound/pulseaudio )
+	python? (
+		${PYTHON_DEPS}
+		dev-python/lxml
+		dev-python/mysql-python
+		dev-python/urlgrabber
+		dev-python/future
+		dev-python/requests-cache
+	)
+	systemd? ( sys-apps/systemd:= )
+	vaapi? ( x11-libs/libva:=[opengl] )
+	vdpau? ( x11-libs/libvdpau )
+	vpx? ( <media-libs/libvpx-1.7.0:= )
```

Comment:
Oh also are you SURE about this? This is going to cause a lot of trouble for people with up-to-date systems.
Check with ldd and tools a-like please.

Reply:
I removed the restriction since the gentoo stable version in within the limitation.

Reply:
Stable vpx is `<media-libs/libvpx-1.7.0:=` so I removed the restriction.

> Check with ldd and tools a-like please.

I am not sure what to do with `ldd` so I ran `ldd -u` on the mythbackend binary.

`$ ldd -u /usr/bin/mythbackend`
Unused direct dependencies:
        /usr/lib64/libmythprotoserver-30.so.30
        /lib64/libpthread.so.0
        /usr/lib64/libQt5Gui.so.5
        /usr/lib64/libQt5Network.so.5
        /usr/lib64/libQt5Xml.so.5
        /usr/lib64/libQt5Sql.so.5
        /usr/lib64/libQt5Script.so.5
        /usr/lib/gcc/x86_64-pc-linux-gnu/8.3.0/libgcc_s.so.1

Removing some qt dependencies in the ebuild has no effect on the unused list.